playground/lottie.ts

```diff
@@ -6,34 +6,28 @@ import { OrbitControl } from "@oasis-engine/controls";
 import { Camera, Entity, WebGLEngine } from "oasis-engine";
 import { LottieAnimation } from "@oasis-engine/lottie";
 
-// Create engine object
 const engine = new WebGLEngine("canvas");
+
 engine.canvas.resizeByClientSize();
 
-const scene = engine.sceneManager.activeScene;
-const rootEntity = scene.createRootEntity();
+const root = engine.sceneManager.activeScene.createRootEntity();
 
-// Create camera
-const cameraEntity = rootEntity.createChild("Camera");
-cameraEntity.transform.setPosition(0, 0, 5);
-cameraEntity.addComponent(Camera);
+const cameraEntity = root.createChild("camera");
+const camera = cameraEntity.addComponent(Camera);
+cameraEntity.transform.setPosition(0, 0, 10);
 cameraEntity.addComponent(OrbitControl);
 
-engine.resourceManager
-  .load<Entity>({
-    urls: [
-      "https://gw.alipayobjects.com/os/OasisHub/20759aba-867f-4256-8504-935743240c78/data.json",
-      "https://gw.alipayobjects.com/os/bmw-prod/083ff1ac-15d9-42cb-8d7a-5b7c39b81f5f.json",
-      "https://gw.alipayobjects.com/mdn/rms_e54b79/afts/img/A*Ax4DSrekVhEAAAAAAAAAAAAAARQnAQ"
-    ],
-    type: "lottie"
-  })
-  .then((lottieEntity) => {
-    rootEntity.addChild(lottieEntity);
-    const lottie = lottieEntity.getComponent(LottieAnimation);
-    lottieEntity.transform.setScale(1.5, 1.5, 1);
-    lottie.isLooping = true;
-    lottie.play();
-  });
+engine.resourceManager.load<Entity>({
+  urls: [
+    'https://gw.alipayobjects.com/os/bmw-prod/9ad65a42-9171-47ab-9218-54cf175f6201.json',
+    'https://gw.alipayobjects.com/os/bmw-prod/58cde292-8675-4299-b400-d98029b48ac7.atlas',
+  ],
+  type: 'lottie'
+}).then((lottieEntity) => {
+  root.addChild(lottieEntity);
+  const lottie:LottieAnimation = lottieEntity.getComponent(LottieAnimation);
```

Comment:
Use `const lottie = lottieEntity.getComponent(LottieAnimation);` instead of
`const lottie:LottieAnimation = lottieEntity.getComponent(LottieAnimation);`